src/Core/src/Platform/Windows/RootNavigationView.cs

```diff
@@ -22,13 +22,18 @@ public partial class RootNavigationView : MauiNavigationView
 
 		public RootNavigationView()
 		{
-			InitializeComponent();
+			IsSettingsVisible = false;
+			IsPaneToggleButtonVisible = false;
+			PaneDisplayMode = NavigationViewPaneDisplayMode.LeftMinimal;
+			IsTitleBarAutoPaddingEnabled = false;
+			IsBackButtonVisible = NavigationViewBackButtonVisible.Collapsed;
```

Comment:
Set the xaml values in C# to avoid errors so ambiguous that you feel like a -10x engineer. If there is any error at any point in the code or types inside this constructor, the whole operation fails with "Xaml Parser Failed" and no way of knowing.